common/address.go

```diff
@@ -48,6 +48,19 @@ func (a Address) InternalAddress() (InternalAddress, error) {
 	return *internal, nil
 }
 
+func (a Address) InSameSliceAs(cmp Location) bool {
+	loc := *a.Location()
```

Comment:
I think we should change the Location() function to panic instead of returning nil if the location can't be found. It should be impossible so it's ok to panic, better than a nil pointer error.

Reply:
No, because someone could produce a block which contained an ETX destined to an illegal destination, and it would cause our nodes to panic. We need to be able to reject such blocks without crashing.

Reply:
@wizeguyy I'm pretty sure our address sharding ranges cover the entire possible address space. In any case this function was removed so I'm resolving this.